test/unit/api/emit.js

```diff
@@ -7,10 +6,9 @@ describe('api/emit', function () {
   var triggered;
 
   beforeEach(function () {
-    fixture('<div><span></span></div>');
-
-    child = fixture().querySelector('span');
-    parent = fixture().querySelector('div');
+    child = document.createElement('child');
```

Comment:
We don't have to append the test element to a fixture anymore because we are simulating bubbling for trees not in the document.